extensions/tokenizers/src/main/java/ai/djl/huggingface/translator/TextEmbeddingTranslatorFactory.java

```diff
@@ -62,8 +64,17 @@ public <I, O> Translator<I, O> newInstance(
                             .optTokenizerPath(modelPath)
                             .optManager(model.getNDManager())
                             .build();
-            TextEmbeddingTranslator translator =
-                    TextEmbeddingTranslator.builder(tokenizer, arguments).build();
+            TextEmbeddingTranslator.Builder builder =
+                    TextEmbeddingTranslator.builder(tokenizer, arguments);
+
+            String mode = Utils.getEnvOrSystemProperty("DJL_SENTENCE_TRANSFORMER_MODE");
```

Comment:
Why is this an env variable and not an argument or model property? Because this mode seems to be local to the model rather than apply to all models.

Reply:
Our current model zoo model doesn't apply this Dense layer. If we modify the serving.properties in the existing model zoo model, will break existing behavior. With this env var, we can enable new behavior without impact existing user.